nextflow_schema.json

```diff
@@ -142,41 +149,41 @@
             "properties": {
                 "trunclenf": {
                     "type": "integer",
+                    "default": 280,
                     "description": "DADA2 read truncation value for forward strand, set this to 0 for no truncation",
                     "help_text": "Read denoising by DADA2 creates an error profile specific to a sequencing run and uses this to correct sequencing errors. This method prefers when all reads to have the same length and as high quality as possible while maintaining at least 20 bp overlap for merging. One cutoff for the forward read `--trunclenf` and one for the reverse read `--trunclenr` truncate all longer reads at that position and drop all shorter reads.\nIf not set, these cutoffs will be determined automatically for the position before the mean quality score drops below `--trunc_qmin`.\n\nFor example:\n\n```bash\n--trunclenf 180 --trunclenr 120\n```\n\nPlease note:\n\n1. Overly aggressive truncation might lead to insufficient overlap for read merging\n2. Too little truncation might reduce denoised reads\n3. The code choosing these values automatically cannot take the points above into account, therefore checking read numbers is essential",
                     "fa_icon": "fas fa-ban"
                 },
                 "trunclenr": {
                     "type": "integer",
+                    "default": 250,
```

Comment:
these trunc values will likely change depending on the quality of our dataset, but hopefully not by too much!